bl/drawable/inc/drawable.hpp

```diff
@@ -3,24 +3,28 @@
 #include <limits>
 #include <list>
 #include <memory>
-#include <tuple>
 #include <unordered_map>
-#include <utility>
+
+// TODO temp struct, will be removed in the next PR
+struct Color {
+    int r, g, b, a;
```

Comment:
shouldn't be in this PR it is not fixing any clang-tidy issue

Reply:
fixed